AxonDeepSeg/visualization/visualize.py

```diff
@@ -1,174 +1,218 @@
-import pickle
+import os
 import sys
-from sys import platform as _platform
-if 'pytest' in sys.modules:
-    import matplotlib as mpl
-    mpl.use('Agg') # Enforces mpl to not open new plot windows
-elif _platform == "darwin": # Mac OSX
-    import matplotlib as mpl
-    mpl.use('TkAgg')
-import matplotlib.pyplot as plt
-from scipy.misc import imread
+import pickle
+
+# Scientific modules imports
+import numpy as np
 from sklearn.metrics import accuracy_score
-from ..testing.segmentation_scoring import score_analysis, dice
 from sklearn import preprocessing
-import os
+from scipy.misc import imread
 from tabulate import tabulate
-import numpy as np
+
+# Graphs and plots imports
+from matplotlib.backends.backend_agg import FigureCanvasAgg as FigureCanvas
+from matplotlib.figure import Figure
+
+# AxonDeepSeg imports
 import AxonDeepSeg.ads_utils
+from ..testing.segmentation_scoring import score_analysis, dice
 
 
-def visualize_training(path_model, start_visu=0):
+def visualize_training(path_model, start_for_viz=0):
```

Comment:
@maf88 any particular reason why you renamed `start_visu` to `start_for_viz`? Not super attached to either, just curious

Reply:
`start_viz`seems  like a boolean flag. `start_for_viz` seemed more expressive, but I will change it to `viz_start_point`or `viz_start_frame` or whatever fits